game/src/ecs/components/stats/StatsComponent.java

```diff
@@ -0,0 +1,36 @@
+package ecs.components.stats;
+
+import ecs.components.Component;
+import ecs.entities.Entity;
+
+public class StatsComponent extends Component {
+
+    private DamageModifier damageModifier = new DamageModifier();
+
+    /**
+     * Create a new component and add it to the associated entity
+     *
+     * @param entity associated entity
+     */
+    public StatsComponent(Entity entity) {
+        super(entity);
+    }
+
+    /**
+     * Get the stats object of the entity
```

Comment:
```suggestion
     * Get the DamageModifier object of the entity
```
Öfters steht noch im Doc der alte Name `stats` obwohl dies nun `DamageModifiers` heißt.